### PR TITLE
pkp/pkp-lib#1707 add supplementary file support for the native import…

### PR DESCRIPTION
**Pull request by bozana**
… and export

s. https://github.com/pkp/pkp-lib/issues/1707

### CODE REVIEW

File: plugins/importexport/native/filter/NativeXmlMonographFilter.inc.php [hunk 1]
```diff
@@ -76,6 +76,9 @@ function handleChildElement($n, $submission) {
 			case 'artwork_file':
 				$this->parseSubmissionFile($n, $submission);
 				break;
+			case 'supplementary_file':
+				$this->parseSubmissionFile($n, $submission);
+				break;
```
Comment (NateWr) [inline plugins/importexport/native/filter/NativeXmlMonographFilter.inc.php]: The two `case` statements can be combined again.

File: plugins/importexport/native/filter/NativeXmlSupplementaryFileFilter.inc.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+	//
+	// Extend functions in the parent class
+	//
+	/**
+	 * Handle a child node of the submission file element; add new files, if
+	 * any, to $submissionFiles
+	 * @param $node DOMElement
+	 * @param $fileId int File id
+	 * @param $stageId int SUBMISSION_FILE_...
```
Comment (NateWr) [inline plugins/importexport/native/filter/NativeXmlSupplementaryFileFilter.inc.php]: This `@param` doc should be one line up.

File: plugins/importexport/native/filter/NativeXmlSupplementaryFileFilter.inc.php [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+			$submissionFiles[count($submissionFiles)-1]->$setterFunction($value, $locale);
+		} else switch ($node->tagName) {
+			case 'date_created':
+				$submissionFiles[count($submissionFiles)-1]->setDateCreated(strtotime($node->textContent));
+				break;
+			case 'language':
+				$submissionFiles[count($submissionFiles)-1]->setLanguage($node->textContent);
+				break;
+			default:
+				return parent::handleChildElement($node, $stageId, $fileId, $submissionFiles);
```
Comment (NateWr) [inline plugins/importexport/native/filter/NativeXmlSupplementaryFileFilter.inc.php]: `return` here can be removed, I think...

**Comment by bozana** [comment]
miraculously, the tests are failing... :-O ;-)

**Comment by NateWr** [comment]
You may need to rebase your `pkp-lib` submodule commit. These are all linting errors which were fixed in #1536.

**Comment by bozana** [comment]
Ýes, now the tests passed... :-) @NateWr, could you then review the code changes after the first review...

**Pull request merged**